augly/video/functional.py

```diff
@@ -706,8 +706,7 @@ def hflip(
     """
     func_kwargs = helpers.get_func_kwargs(metadata, locals(), video_path)
 
-    hflip_aug = af.VideoAugmenterByHFlip()
-    vdutils.apply_ffmpeg_augmenter(hflip_aug, video_path, output_path)
+    af.VideoAugmenterByHFlip().add_augmenter(video_path, output_path)
```

Comment:
```suggestion
    hflip_aug = af.VideoAugmenterByHFlip()
    hflip_aug.add_augmenter(video_path, output_path)
```
very small nit: for other augmenters we'll want to do it more like this since there will be kwargs so putting it in one line could get messy.